qrcode/main.py

```diff
@@ -27,12 +27,13 @@ def _check_box_size(size):
 class QRCode:
     def __init__(self, version=None,
                  error_correction=constants.ERROR_CORRECT_M,
-                 box_size=10, border=4,
+                 box_size=50, border=1,
```

Comment:
This is not a good idea. 3rd party code may rely on the standard values ```box_size=10``` and ```border=4```
If you change the standard values users will get unexpected results